lib/gssapi/spnego/accept_sec_context.c

```diff
@@ -888,12 +897,11 @@ acceptor_continue
 			      input_chan_bindings,
 			      &obuf,
 			      delegated_cred_handle);
-	    if (ret == GSS_S_COMPLETE || ret == GSS_S_CONTINUE_NEEDED) {
-		mech_output_token = &obuf;
-	    }
+	    mech_output_token = &obuf;
 	    if (ret != GSS_S_COMPLETE && ret != GSS_S_CONTINUE_NEEDED) {
 		free_NegotiationToken(&nt);
-		send_reject(&junk, output_token);
+		send_reject(&junk, mech_output_token, output_token);
+		gss_release_buffer(&junk, mech_output_token);
```

Comment:
This fixes a leak!

Reply:
Yes, I sneaked that in. Probably should have been a separate PR.

Reply:
Nah, it's ok here.